docs/setup/connect-to-elasticsearch.asciidoc

```diff
@@ -1,91 +1,98 @@
 [[connect-to-elasticsearch]]
-== Add your data
-++++
-<titleabbrev>Add data</titleabbrev>
-++++
+== Add data
 
-To start working with your data in {kib}, use one of the many ingest options,
-available from the home page.
-You can collect data from an app or service,
-or upload a file that contains your data.  If you're not ready to use your own data,
-add a sample data set and give {kib} a test drive.
+The best way to add data to the Elastic Stack is to use one of our many integrations.
+All integrations are available in a single view, and
+{kib} guides you there from the Welcome screen, home page, and main menu.
 
 [role="screenshot"]
-image::images/addData_home_7.15.0.png[Built-in options for adding data to Kibana:  Add data, Add Elastic Agent, Upload a file]
+image::images/add-integration.png[Integrations page from which you can choose integrations to start collecting and analyzing data]
 
-[float]
-[[add-data-tutorial-kibana]]
-=== Add data
-
-Want to ingest logs, metrics, security, or application data?
-Install and configure a Beats data shipper or other module to periodically collect the data
-and send it to {es}. You can then use the pre-built dashboards to explore and analyze the data.
-
-[role="screenshot"]
-image::images/add-data-tutorials.png[Add Data tutorials]
+NOTE: If an integration is available for both
+https://www.elastic.co/guide/en/fleet/master/beats-agent-comparison.html[Elastic Agent and Beats], the *Integrations* view defaults to the
+Elastic Agent integration, if it is generally available (GA).
+To switch to a
+Beats integration, use the filter on the bottom left.
 
-[discrete]
-=== Add Elastic Agent
+[float]
+=== Add data with Elastic solutions
+
+A good place to start is with one of the promoted integrations:
+
+* *Web site search crawler.*
+Discover, extract, and index your web content into App Search engines using the
+{app-search-ref}/web-crawler.html[Enterprise Search web site crawler].
+Search across Google Drive, GitHub, Salesforce, and many other web services using
+{workplace-search-ref}/workplace-search-content-sources.html[Workplace Search content sources].
+
+* *Elastic APM.*
+Get logs, metrics, traces, and uptime data into the Elastic Stack.
+Integrations are available for popular services and platforms,
+such as Nginx, AWS, and MongoDB,
+and generic input types like log files.
+Refer to {observability-guide}/observability-introduction.html[Elastic Observability]
+for more information.
+
+* *Endpoint Security.*
+Protect your hosts and send logs, metrics, and endpoint security data
+to Elastic Security.
+Refer to {security-guide}/ingest-data.html[Ingest data to Elastic Security]
+for more information.
 
-*Elastic Agent* is the next generation of
-data integration modules, offering
-a centralized way to set up your integrations.
+[float]
+=== Add data with language clients
```

Comment:
The title uses `language clients`, but the sentence below uses `programming language`. To avoid confusion, I would pick one.